buildscripts/kokoro/windows.bat

```diff
@@ -0,0 +1,40 @@
+@rem ##########################################################################
+@rem
+@rem Script to set up Kokoro worker and run Windows tests
+@rem
+@rem ##########################################################################
+
+@rem Enter repo root
+cd /d %~dp0\..\..
+
+set WORKSPACE=T:\src\github\grpc-java
+set ESCWORKSPACE=%WORKSPACE:\=\\%
+
+mkdir grpc-java-helper
+
+@rem Install 7za
+wget http://www.7-zip.org/a/7za920.zip
+unzip 7za920.zip
```

Comment:
If we have unzip then we don't need 7za. Our other build machines don't have unzip. But it'd be annoying to support both in make_dependencies.bat. Add a TODO here to swap make_dependencies.bat to use unzip instead.